wg-manifests/charter.md

```diff
@@ -1,70 +1,32 @@
-# WG Manifests Charter
+# WG Platform/Manifests Charter
 
-This charter adheres to the conventions, roles and organization management
-outlined in [wg-governance].
+This platform/manifests charter describes the working mode / reality / status quo of the last 5 years as of March 2025.
+It tries to be as lean as possible and balance community and commercial interests.
 
 ## Scope
 
-- Provide a catalog (centralized repository) of Kubeflow application manifests.
-- Provide a catalog of third-party apps for common services.
-
-### In scope
-
-#### Code, Binaries and Services
```

Comment:
I don't understand why did you remove this part from the charter.
The goal of the WGs is to own the code, and looking at your updates it is not clear what this WG owns
https://github.com/kubeflow/community/blob/a5401ba5362e05850f2029b9e461fbeb663c93af/wgs/overview.md#background

Reply:
Well next to the manifests adjusted to work in a multi-tenancy platform (configuration) and the dependencies we have the documentation, synchronization and probably the largest part our integration tests that make sure that the components work end-to-end together as multi-tenant platform. Maybe i missed something.

Reply:
I switched to the old structure and clarified it